presto-spi/src/main/java/com/facebook/presto/spi/type/FloatType.java

```diff
@@ -35,13 +35,13 @@ private FloatType()
     @Override
     public boolean isComparable()
     {
-        return false;
+        return true;
```

Comment:
Oh I see, you changed this in this commit. I would merge this with the one that introduced `FloatType` because it's confusing that it doesn't return `true` for these